### DISCUSSION TITLE
Files not showing in Files App

### DISCUSSION THREAD

**Discussion by ericklindberg** [MobiusSync/MobiusSync#114] [Q&A] [answered]
I’ve got files syncing, and they show as synced in the Mobius sync app, however they are not visible in the Files app. I’m using the sync with other app sandbox feature. Any help is appreciated as I purchased pro specifically for this single use. 
![IMG_4551](https://github.com/MobiusSync/MobiusSync/assets/82782922/19567bf4-29ac-4f0b-af1e-c813bf9e24af)
![IMG_4550](https://github.com/MobiusSync/MobiusSync/assets/82782922/df3ccb98-0483-411d-8b16-670cd06b95a9)

**Comment by SimonPickup** [comment]
Please try a phone reboot first. This often helps with files not showing in Files.

**Comment by ericklindberg** [comment, marked as answer]
I’m going to close this, I found the answer with some additional investigation. If you set the file path for external, then change the name, the file path changes to the name as if it’s keeping it in the Möbius sandbox. Hope this helps somebody down the road.
